CHANGES.txt

```diff
@@ -1,3 +1,65 @@
+2016-09-23 version 3.1.0 (C++/Java/Python/PHP/Objective-C)
+  General
+  * Proto3 support in PHP.
+  * Various bug fixes.
+
+  C++
```

Comment:
Please incorporate haberman@'s comment into this change log:
https://github.com/google/protobuf/pull/2153